*** Makefile	2018-09-19 17:05:59.067745666 +0200
--- Makefile~	2018-09-19 17:07:06.980472539 +0200
*************** PROFILE        := NO
*** 15,21 ****
  
  # TINYXML_USE_STL can be used to turn on STL support. NO, then STL
  # will not be used. YES will include the STL files.
! TINYXML_USE_STL := NO
  
  #****************************************************************************
  
--- 15,21 ----
  
  # TINYXML_USE_STL can be used to turn on STL support. NO, then STL
  # will not be used. YES will include the STL files.
! TINYXML_USE_STL := YES
  
  #****************************************************************************
  
*************** RELEASE_CFLAGS   := -Wall -Wno-unknown-p
*** 31,37 ****
  LIBS		 :=
  
  DEBUG_CXXFLAGS   := ${DEBUG_CFLAGS} 
! RELEASE_CXXFLAGS := ${RELEASE_CFLAGS}
  
  DEBUG_LDFLAGS    := -g
  RELEASE_LDFLAGS  :=
--- 31,37 ----
  LIBS		 :=
  
  DEBUG_CXXFLAGS   := ${DEBUG_CFLAGS} 
! RELEASE_CXXFLAGS := ${RELEASE_CFLAGS} -fPIC
  
  DEBUG_LDFLAGS    := -g
  RELEASE_LDFLAGS  :=
